src/fileprovider.cs

```diff
@@ -316,6 +323,12 @@ interface NSFileProviderDomain {
 		[Export ("hidden")]
 		bool Hidden { [Bind ("isHidden")] get; set; }
 
+		[Advice ("This API is not available when using UIKit on macOS.")]
```

Comment:
should not be needed anymore (and it's not on other members)

Reply:
@spouliot does it hurt to keep it as is (the rest of the old code has it)? or do you want me to go and remove them?

I may also need to remove this from Objective-Sharpie since it is coming from there.

Reply:
they can all be removed together, later
as for sharpie there's other stuff like using the short forms `[NoMacCatalyst]` and `[MacCatalyst (x,y)]` but that might be best to wait to see if other (like dotnet related) changes are also needed